core/domain/story_domain_test.py

```diff
@@ -232,13 +232,22 @@ def setUp(self):
         self.signup('user1@example.com', 'user1')
 
     def _assert_validation_error(self, expected_error_substring):
-        """Checks that the story passes validation."""
+        """Checks that the story passes validation.
+
+        Args:
+            expected_error_substring: string. String that should be a substring of the expected error message.
```

Comment:
We generally try and wrap lines at around 80 characters. This makes it easier for people working on the command line to view diffs. In this case this might look like this:

```suggestion
            expected_error_substring: string. String that should be a substring
                of the expected error message.
```

Reply:
@nadintamer these long lines caused the lint failures on circle CI. The test output is [here](https://circleci.com/gh/oppia/oppia/21381?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link).

Reply:
Done!